### DISCUSSION TITLE
Creality K1 Max & Co Print

### DISCUSSION THREAD

**Discussion by 25glen** [mainsail-crew/mainsail#2091] [Help]
Hi Guys,
Just installed all the hardware for Co Print to my K1 Max, Co Print is a AMS that can be added to any printer, in this process you change the print head to a completely new type with a different connection. However the printer is still looking for the original head (no longer use the print head cable to motherboard) and throws up error key111 on the panel and error key344 in mainsail, any advise on how to disable the printer looking for the original print head would be amazing.

**Comment by mryel00** [comment]
Before you write anything about this not being resolved. You can pay us the same amount that you paid for the printer+Co Print, and we might be willing to help you. Mainsail is just showing you the error, thrown by the custom Klipper of either the K1 Max or Co Print.
Those error keys are not native Klipper and we cannot even say if it's the K1 or the Co Print throwing it. So write the support of one of the companies you bought the hardware from.

**Comment by meteyou** [comment]
All keyXX errors are Creality error codes. You cannot modify a creality printer with something like this, because creality has a "half closed" ecosystem.

So pls ask the creality support for questions like this, but to be honest, I don't think you will get that working with a creality os.
